### PR TITLE
Remove turbolinks completely

### PR DESCRIPTION
**Pull request by vasya539**
- [x] merged with the latest master branch
- [x] resolved all the conflicts, PR could be merged
- [ ] new code has new specs
- [ ] for new models - new factories were added too
- [x] unit specs are passed
- [x] code is well-formatted, Ruby, CSS, etc
- [x] screenshots of new pages, features are added
- [x] code passed internal team code review
- [ ] no instance variables in partials
- [ ] code is well formatted and has spaces!!!

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by vasya539** [comment]
image

File: app/assets/config/manifest.js [hunk 1]
```diff
@@ -19,8 +19,9 @@
 //= link vertical-menu-light.js
 //= link bootstrap.js
 //= link show_file_name_in_file_field.js
-//= link ckeditor/application.js
 //= link jqBootstrapValidation.js
+// = link ckeditor/config.js
```
Comment (igorkasyanchuk) [inline app/assets/config/manifest.js]: а тут чого по іншому код виглядає ніж нижче

Comment (igorkasyanchuk) [reply app/assets/config/manifest.js]: я про = і пробєл

File: config/database.yml [hunk 2]
```diff
@@ -80,6 +80,6 @@ test:
 #
 production:
   <<: *default
-  database: remote_flow_production
-  username: remote_flow
-  password: <%= ENV['REMOTE_FLOW_DATABASE_PASSWORD'] %>
+  database: remote_flow
```
Comment (igorkasyanchuk) [inline config/database.yml]: на проді як база називається?

File: config/initializers/slack.rb [hunk 3]
```diff
@@ -1,2 +1,2 @@
-SLACK_CONFIG = YAML.load_file('config/slack.yml')[Rails.env].symbolize_keys
+SLACK_CONFIG = YAML.load_file('config/slack.yml')[Rails.env].symbolize_keys rescue nil
```
Comment (igorkasyanchuk) [inline config/initializers/slack.rb]: забери  rescue nil

File: config/deploy/production.rb [hunk 4]
```diff
@@ -7,6 +7,7 @@
 # server "example.com", user: "deploy", roles: %w{app web}, other_property: :other_value
 # server "db.example.com", user: "deploy", roles: %w{db}
 
+config.serve_static_files = true
```
Comment (igorkasyanchuk) [inline config/deploy/production.rb]: а це чого?

**Pull request merged**